google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/BulkMutation.java

```diff
@@ -114,6 +115,21 @@ public MutateRowsRequest toProto(RequestContext requestContext) {
         .build();
   }
 
+  /**
+   * Wraps the protobuf {@link MutateRowsRequest}.
+   *
+   * <p>WARNING: Please note that the table name will be overwritten by the configuration in the
+   * BigtableDataClient. The mutation must always be idempotent because it would be retried.
```

Comment:
`The mutation must always be idempotent because it would be retried.` needs to be explained more, please add a similar warning as described below for RowMutation.